circuit_database/halo2/axiom-v0.3.0/mainnet_header/Cargo.toml

```diff
@@ -0,0 +1,73 @@
+[package]
```

Comment:
You have added the mainnet header circuit but we previously did not include this in sample-data.  If it was consciously done, that is fine by me but you should then remove the georli header circuit.  They are the same exact circuit code with different config files

Reply:
How about I remove goerli and make a comment in the README.md of mainnet with the sindri.json to use goerli instead?

Reply:
Acceptable!